commands/update.sh

```diff
@@ -17,7 +17,8 @@ args=(
   "--volume" "$PLUGIN_DIR/update:/update"
   "--workdir" "/bundle_update"
   "--env" "BUNDLE_APP_CONFIG=/bundle_app_config"
-  "--env" "PRE_BUNDLE_UPDATE=${script}"
+  "--env" "PRE_BUNDLE_UPDATE=${pre_bundle_update}"
+  "--env" "POST_BUNDLE_UPDATE=${post_bundle_update}"
```

Comment:
We run the bundle update in a Docker container, so we'll need to pass the argument through via an environment variable. Just like the `PRE_BUNDLE_UPDATE`.